### ISSUE TITLE
Make default new file properties optional

### ISSUE THREAD

**Issue by ElormCodes1** [srg-kostyrko/obsidian-journal#39]
All the four default properties that are automatically added to a new journal file can be made optional so a user can opt out, better still, the file can be totally blank, the properties conflict with other plugins and some custom scripts. thanks.

**Comment by srg-kostyrko** [comment]
The plugin is build around using that properties to know about note (what journal and what dates). So it is not possible to make them optional without major rewrite.

Can you please provide more info on what problems properties are causing. There were issue with Templater but they are fixed in 1.3.0

**Comment by ElormCodes1** [comment]
I use a plugin that automatically adds my tasks to my kanban board but when I do not specify a date for the task, it picks the date of the daily journal as the task date which sometimes is not what I want, I will want the task to be undated but it just takes the date of the journal.

**Comment by srg-kostyrko** [comment]
What plugin are you using? is it CardBoard? or some other?

**Comment by ElormCodes1** [comment]
journals

**Comment by srg-kostyrko** [comment]
no, I meant - which plugin do you use for kanban that has issues with properties that journals plugin is adding?
